pinc/LPage.inc

```diff
@@ -117,7 +117,7 @@ function get_available_page_array(
         }
     } else {
         // The empty/null (i.e., no restriction) case has already been dealt with.
-        die("Bad value for 'preceding_proofer_restriction': '$preceding_proofer_restriction'.");
+        throw new Exception("Bad value for 'preceding_proofer_restriction': '$preceding_proofer_restriction'.");
```

Comment:
Your changes should remove all uses of `abort()` so we can remove the include() as well.

Reply:
Done this and used more specific exceptions.

Reply:
`ValueError` would probably be a better exception to throw here but it isn't available until PHP 8. I've opened a PR to backport it into our code (I'm using it in my project validation work): https://github.com/DistributedProofreaders/dproofreaders/pull/742

Reply:
> `ValueError` would probably be a better exception to throw here but it isn't available until PHP 8. I've opened a PR to backport it into our code (I'm using it in my project validation work): #742

I guess I should wait for this then rebase it.